packages/form-js-editor/src/render/components/properties-panel/entries/index.js

```diff
@@ -8,4 +8,5 @@ export { default as KeyEntry } from './KeyEntry';
 export { default as LabelEntry } from './LabelEntry';
 export { default as TextEntry } from './TextEntry';
 export { default as ValueEntry } from './ValueEntry';
-export { default as CustomValueEntry } from './CustomValueEntry';
\ No newline at end of file
+export { default as CustomValueEntry } from './CustomValueEntry';
+export * from './ValuesSourceEntries';
```

Comment:
Is this expected? It now also exports some helpers as `getValuesSource`, which I wouldn't expect to import from `entries`. What I'd rather expect

```js
import { getValuesSource, VALUES_SOURCES } from './entries/ValuesSourceEntries';
```

Reply:
Going to split all the assisting code into its own util, so that it may be used by entries and groups without issue